src/core/_core-maps.scss

```diff
@@ -9,3 +9,5 @@
 @import "maps/core-shadows-map";
 @import "maps/core-spacing-map";
 @import "maps/core-typography-map";
+//
```

Comment:
is this `//` intentional?

Reply:
yes, it's an exception, common scope is a small anti pattern, just wanted a small separation. 
its a common thing in SCSS loaders to separate using a line break or // 
i can add a line break if this is more clear

Reply:
> It looks good to me (especially the stylelint rule 👍 ), but I think I'm missing a bit of context to approve it..

The Task is attached.
The disabled colors (text,background) where defined with Opacity, now we corrected this. 
added new color tokens and reassigned the disabled tokens with the new values.
currently still in the old method. soon to be replaced.

just a routine token update.

Reply:
> yes, it's an exception, common scope is a small anti pattern, just wanted a small separation. its a common thing in SCSS loaders to separate using a line break or // i can add a line break if this is more clear

I think I understand. Maybe it's just me, but it was a bit confusing. A line break would have been more intuitive for me :) but that's up to you

Reply:
line breaks can be accidental. leaving it for the time being.
when we will have some luxury time will can define SCSS commenting and related standards.